src/Parser/PHP/ConsumedSymbolCollector.php

```diff
@@ -56,6 +58,12 @@ public function enterNode(Node $node)
 
     public function getSymbolNames(): array
     {
+        foreach ($this->strategies as $strategy) {
```

Comment:
Technically this implementation and fix works.

But logically composer-unused always uses all strategies, so this loop will add "some" performance impact as it iterates over all strategies.

See: https://github.com/composer-unused/composer-unused/blob/main/config/services.php#L68-L84

Reply:
I understand !